config/babel.default.config.js

```diff
@@ -1,7 +1,8 @@
 const { getCssClassNames } = require("./utils");
 
 const reactCssPluginOptions = {
-  generateScopedName: getCssClassNames()
+  generateScopedName: getCssClassNames(),
+  autoResolveMultipleImports: true
```

Comment:
[docs](https://www.npmjs.com/package/babel-plugin-react-css-modules)
`Allow multiple anonymous imports if styleName is only in one of them.`
so that we can do this while importing arrow components >>
![Screenshot 2021-02-13 at 8 34 36 PM](https://user-images.githubusercontent.com/27222520/107853284-fc995d80-6e3a-11eb-822d-a10930caa04b.png)